### DISCUSSION TITLE
V2ray建立连接后,测速正常,能上部份外国网站,打不来youtube及google等,请大神指教下.

### DISCUSSION THREAD

**Discussion by jm-king** [v2fly/v2ray-core#2508] [General]
V2ray建立连接后,测速正常,能上部份外国网站,打不开youtube及google等,请大神指教下.

我在HK放了一台windows的主机来运行V2ray.在国内连接HK的v2ray后测速有11M.上国外网站时只有部分能上(一直能上,视频网站一直播放,几个网站一起播放都没问题),但google及youku等出现断流问题(不定时能正常打开,过几十秒后又不行),本人对网络有点小白,请大神们指教下,

以上情况是2023年5月12日的前几天发现(不超过一周,确实那天记不起了).

服务端配置文件如下:
{
  "inbounds": [
    {
      "port": 443,     
      "protocol": "vmess",
      "settings": {
        "clients": [
          {
            "id": "UUID我就不显示了",
            "alterId": 0
          }
        ]
      },
      "streamSettings": {
        "network": "ws",
        "wsSettings": {
        "path": "/aaa"
        }
      }
    }
  ],
  "outbounds": [
    {
      "protocol": "freedom",
      "settings": {}
    }
  ]
}

**Comment by dimjoker** [comment]
可能是DNS污染了，可以试试添加dns及router配置试试

**Comment by qianbinbin** [comment]
我最近疑似遭遇了严重的 SNI 阻断，域名本身的网站可以正常访问，但是油管、GitHub 等网站断断续续
方案用过 VLESS+WS+TLS 和 VMess+WS+TLS，套不套 CDN 都一样

**Reply by jm-king** [reply]
你看看是不是移动的线路.

**Reply by qianbinbin** [reply]
> 你看看是不是移动的线路.

电信

**Comment by jm-king** [comment]
谢谢以上两位的回复，现在得知的情况是不论是自己建的还是外国一键生成的VPS都有这问题，通几十秒断几十秒，这问题出在移动线路，我用有问题的VPS，在移动线路再VPN到电信线路到现在还没发现这情况（有24小时以上了）。

**Comment by om2mo** [comment]
看上去你只有裸的Websocket，这并不安全和稳定

**Reply by jm-king** [reply]
这问题我也知道,因TLS有点复杂,我小白还没搞会,但用国外的一键脚 本生成的有WS+TLS的也是一样,断流时都是同时的.

**Reply by om2mo** [reply]
你需要一个域名并用web做反代，这是长期稳定的方案

**Reply by qianbinbin** [reply]
> 你需要一个域名并用web做反代，这是长期稳定的方案

已经不稳定了，我就是这么干的，用了几年，最近出现严重的阻断
